ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/eos_cli_config_gen.schema.yml

```diff
@@ -93,22 +93,47 @@ keys:
           description: 'Action as string
 
             Example: "permit GSHUT 65123:123"'
+  interface_groups:
+    type: list
+    primary_key: name
+    convert_types:
+    - dict
+    display_name: Interface Groups
+    items:
+      type: dict
+      keys:
+        name:
+          type: str
+          required: true
+          display_name: Group-name
+        interfaces:
+          type: list
+          items:
+            type: str
+        bgp_maintenance_profiles:
+          type: list
+          items:
+            type: str
+        interface_maintenance_profiles:
+          type: list
+          items:
+            type: str
   ip_community_lists:
     type: list
     secondary_key: entries
     primary_key: name
     convert_types:
     - dict
     display_name: IP Community Lists
-    description: "AVD supports 2 different data models for community lists:\n\n- The
-      legacy `community_lists` data model that can be used for compatibility with
-      the existing deployments.\n- The improved `ip_community_lists` data model.\n\nBoth
-      data models can coexist without conflicts, as different keys are used: `community_lists`
-      vs `ip_community_lists`.\nCommunity list names must be unique.\n\nThe improved
-      data model has a better design documented below:\n\ncommunities and regexp MUST
-      not be configured together in the same entry\npossible community strings are
-      (case insensitive):\n - GSHUT\n - internet\n - local-as\n - no-advertise\n -
-      no-export\n - <1-4294967040>\n - aa:nn\n"
+    description: "AVD supports 2 different data models for community lists:\n\n- The\
```

Comment:
I am not sure why these changes show up in this PR. Maybe it is something with versions? Please check `ansible --version` if you have `libyaml` installed.

Reply:
I approved the PR, but please still look into this. We might have to implement some changes to the schema generation to avoid such changes

Reply:
yes sure, I am looking into the issue.

Reply:
I think it is the ansible version more than the python version making this difference. I run py3.10 as well.